crates/re_space_view_spatial/src/lib.rs

```diff
@@ -75,3 +77,57 @@ fn query_pinhole(
                 .map(|c| c.value),
         })
 }
+
+pub(crate) fn background(
+    ctx: &re_viewer_context::ViewerContext<'_>,
+    query: &re_viewer_context::ViewQuery<'_>,
+) -> Option<Background> {
+    let blueprint_db = ctx.store_context.blueprint;
+    let blueprint_query = ctx.blueprint_query;
+    let background = re_space_view::query_space_view_sub_archetype::<Background>(
+        query.space_view_id,
+        blueprint_db,
+        blueprint_query,
+    )
+    .0;
+    background.ok().flatten()
+}
```

Comment:
instead of repeating those property getters over and over again, we should just make a variant that takes ctx, query and the archetype as generic argument and stick it into re_space_view as well